src/Blocks/BlockBigFlower.h

```diff
@@ -18,6 +18,11 @@ class cBlockBigFlowerHandler :
 	{
 	}
 
+	virtual bool DoesIgnoreBuildCollision(cPlayer * a_Player, NIBBLETYPE a_Meta) override
+	{
+		return (a_Meta & E_META_BIG_FLOWER_DOUBLE_TALL_GRASS || a_Meta & E_META_BIG_FLOWER_LARGE_FERN);
```

Comment:
I'd prefer the explicit longer condition:
`((a_Meta & E_META_BIG_FLOWER_DOUBLE_TALL_GRASS) != 0) || ...`
After all, the whole codebase has the style that we don't use integers as booleans, we always use an explicit bool result.